resolver/addressable_resolver.go

```diff
@@ -140,7 +164,6 @@ func (r *URIResolver) URIFromObjectReference(ctx context.Context, ref *corev1.Ob
 	if ref == nil {
 		return nil, apierrs.NewBadRequest("ref is nil")
 	}
-
```

Comment:
I think the newline is worth keeping here, since it separates the guard clause from the rest of the method

Reply:
@SwEngin just waiting on adding this newline back to lgtm this!

Reply:
Done!